frontend/src/UseFormSender.js

```diff
@@ -0,0 +1,31 @@
+import { useState } from 'react';
+import axios from 'axios';
+
+const UseFormSender = (reqType, url) => {
```

Comment:
Also let's change it's name please request are not always done in forms and sender is not saying much. We can make it flexible for all of the api endpoints and as it will always have `use` prefix WDYT if we name it `useAPI` and make the `http://localhost:8080/` a default prefix to the url.(Check the axios related comment)
So it can be used like this `const {...} = useAPI('POST', '/documents');`